landoapi/cli.py

```diff
@@ -52,24 +52,16 @@ def cli():
 
 
 @cli.command()
-@click.option("--init-s3", is_flag=True)
-def init(init_s3):
-    """Initialize Lando API (Create the DB, etc.)"""
-    # Create the database and set the alembic version to
-    # head revision.
-    from landoapi.storage import alembic, db
-
-    db.create_all()
-    alembic.stamp("head")
-
+def init_s3(init_s3):
```

Comment:
If we keep this function you'll need to remove the `init_s3` argument for it to work.